onnxruntime/core/providers/coreml/model/model.mm

```diff
@@ -169,6 +171,60 @@ Status CreateInputFeatureProvider(const std::unordered_map<std::string, OnnxTens
   conversion_buffers_out = std::move(conversion_buffers);
   return Status::OK();
 }
+
+bool IsArrayContiguous(MLMultiArray* array) {
+  int64_t batch_stride = [array.strides[0] longLongValue];
+  const auto* shape = array.shape;
+  int64_t batch_elems = 1;
+  for (unsigned long i = 1; i < shape.count; i++) batch_elems *= [shape[i] longLongValue];
+  return batch_stride == batch_elems;
+}
+
+onnxruntime::common::Status copyMLMultiArrayBuffer(const void* mlmultiarray_buffer, void* tensor_buffer,
```

Comment:
```suggestion
Status CopyMLMultiArrayBuffer(const void* mlmultiarray_buffer, void* tensor_buffer,
```
- function naming convention is `MyFunctionName`
- no need to fully onnxruntime::common::Status here